models/AddressLocation.yaml

```diff
@@ -0,0 +1,71 @@
+type: object
+name: AddressLocation
+x-stoplight:
+  id: s3q6yalatyags
+description: "\t\nAn interface used to express a location using a `UnLocationCode`."
+properties:
+  locationName:
+    type: string
+    format: int64
+    description: The name of the location.
+    pattern: ^\S+(\s+\S+)*$
+    example: Port of Amsterdam
+    maxLength: 100
+  locationType:
+    type: string
+    enum:
+      - ADT
```

Comment:
Is there a reason why this is changed from `ADDR` --> `ADT`

Reply:
Just a typo, good spot!